flow/core/kernel/vehicle/base.py

```diff
@@ -139,21 +139,8 @@ def apply_acceleration(self, veh_id, acc):
             list of vehicle identifiers
         acc : float or array_like
             requested accelerations from the vehicles
-        """
-        raise NotImplementedError
-
-    def apply_acceleration_not_smooth(self, veh_id, acc):
-        """Apply the acceleration requested by a vehicle in the simulator.
-
-        In SUMO, this function applies setSpeed method which doesn't apply
-        smoothing.
-
-        Parameters
-        ----------
-        veh_id : str or list of str
-            list of vehicle identifiers
-        acc : float or array_like
-            requested accelerations from the vehicles
+        smooth : bool
+            whether to apply acceleration smoothly or not, default: True
```

Comment:
nit: this needs to be expanded upon. What it means for an accel to be smooth is not obvious